sunpy/util/sysinfo.py

```diff
@@ -59,10 +59,10 @@ def system_info():
     """
     base_reqs = get_distribution("sunpy").requires()
     base_reqs = {base_req.name.lower() for base_req in base_reqs}
-    extra_reqs = get_distribution("sunpy").requires(extras=["all"])
```

Comment:
@nabobalis Here's where we'll need to figure a workaround, for now I've removed it for PyInstaller's tests to pass.

Reply:
What about replacing the "all" with a list of every section?

Reply:
I'm working on that bug (pyinstaller/pyinstaller#5774) so in theory you should just be able to twiddle your thumbs for a few days...

Reply:
It's merged. We're planning on a 4.3.1 patch release soon so you shouldn't have to install from Github for long. It'd probably be a good idea to stick a PyInstaller version check at the top of the hook though...

Reply:
Thanks again @bwoodsend